source/features/repo-header-info.tsx

```diff
@@ -0,0 +1,63 @@
+import * as pageDetect from 'github-url-detection';
+import {RepoForkedIcon, StarIcon} from '@primer/octicons-react';
+import React from 'dom-chef';
+import {CachedFunction} from 'webext-storage-cache';
+
+import observe from '../helpers/selector-observer.js';
+import features from '../feature-manager.js';
+import api from '../github-helpers/api.js';
+import GetRepositoryInfo from './repo-header-info.gql';
+import {cacheByRepo} from '../github-helpers/index.js';
+import abbreviateNumber from '../helpers/abbreviate-number.js';
+
+type RepositoryInfo = {
+	isFork: boolean;
+	stargazerCount: number;
+};
+
+const repositoryInfo = new CachedFunction('stargazer-count', {
+	async updater(): Promise<RepositoryInfo> {
+		const {repository: {isFork, stargazerCount}} = await api.v4(GetRepositoryInfo);
+
+		return {isFork, stargazerCount};
```

Comment:
Since GQL has exact output, we can use:

```suggestion
		const {repository} = await api.v4(GetRepositoryInfo);
		return repository;
```